src/configProvider.ts

```diff
@@ -23,25 +24,36 @@ interface ConfigInstance {
   MB_FORGOT_PASSWORD_TOKEN_DAYS_TO_LIVE: number;
   SENDGRID_KEY: string;
   SENDGRID_PRINT_ONLY: boolean;
+  PORT: string;
 }
 let instance: ConfigInstance;
 
 export default () => {
   if (!instance) {
-    const envFilePath = getConfig(`MB_ENV_FILE`);
 
+    const envFilePath = getConfig("MB_ENV_FILE");
+    
+    const overrideEnvFilePath = getConfig("MB_ENV_FILE_OVR");
+    
     config({
       path: path.join(__dirname, `..`, envFilePath),
     });
 
+    // override
+    const envConfig = parse(fs.readFileSync(path.join(__dirname, "..", overrideEnvFilePath)));
```

Comment:
what happens if the file does not exist? (this is the default scenario)

Reply:
also, if the dev-override.env file is not present then the values will be picked from dev.env and i have tested that. It works only if the file is present in the project at the specified location mentioned in the readme file

Reply:
done